src/mz/src/api.rs

```diff
@@ -62,22 +62,20 @@ impl FromStr for CloudProviderRegion {
 }
 
 #[derive(Debug, Deserialize, Clone)]
-#[serde(rename_all = "camelCase")]
-pub struct Region {
-    pub environment_controller_url: String,
+pub struct RegionInfo {
+    pub sql_address: String,
+    pub http_address: String,
+    pub resolvable: bool,
 }
 
 #[derive(Debug, Deserialize, Clone)]
-#[serde(rename_all = "camelCase")]
-pub struct Environment {
-    pub environmentd_pgwire_address: String,
-    pub environmentd_https_address: String,
-    pub resolvable: bool,
+pub struct Region {
+    pub region_info: RegionInfo,
```

Comment:
this should be an `Option<RegionInfo>` since the region_info field can be nulled when the environment isn't yet available

Reply:
Yes! I noticed when looking at your pull request. That's the work I'm still pending to do.